app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,223 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class TaskServiceTest {
+    private final long INVALID_TASK_ID = 0L;
+
+    private TaskService taskService;
+
+    @BeforeEach
+    void setUp() {
+        taskService = new TaskService();
+    }
+
+    @Nested
+    @DisplayName("getTasks() 메서드는")
+    class Describe_getTasks {
+
+        @Nested
+        @DisplayName("task가 여러개 있을 때")
+        class Context_with_tasks {
+            @BeforeEach
+            void setUp() {
+                taskService.createTask(new Task(1L, "test1"));
+                taskService.createTask(new Task(2L, "test2"));
+            }
+
+            @Test
+            @DisplayName("모든 task를 리턴한다")
+            void it_returns_tasks() {
+                List<Task> tasks = taskService.getTasks();
+
+                assertThat(tasks.size()).isEqualTo(2);
+            }
+        }
+
+        @Nested
+        @DisplayName("task가 없을 때")
+        class Context_without_task {
+            @Test
+            @DisplayName("task가 없는 경우 빈 리스트를 반환한다")
+            void it_returns_empty() {
+                List<Task> tasks = taskService.getTasks();
+
+                assertThat(tasks.size()).isEqualTo(0);
+            }
+        }
+    }
+
+
+    @Nested
+    @DisplayName("getTask() 메서드는")
+    class Describe_getTask {
+        @Nested
+        @DisplayName("기존에 있는 task의 id로 요청할 때")
```

Comment:
`기존`이라는 단어의 사용은 피하도록 합시다. 당장 오늘 코딩할 때에는 '기존'이 명확하게 느껴질 수 있어요. 그런데 몇년간 관리한 코드에서 여기에서도 '기존'이 있고 저기에서도 '기존'이 있고 그러면 골치아파집니다. 기존은 시간에 의존하는 표현이기 때문에 기존이란 단어를 사용하기 시작해서 누적되면 전체적으로 이해를 해치는 힌트가 됩니다. 

```suggestion
        @DisplayName("저장되어 있는 task의 id가 주어지면")
```

Reply:
감사합니다! 
041e434c3ec27a44fa47d25135ef34f23b4bd997
변경했습니다!!